lib/simplecov/result_merger.rb

```diff
@@ -116,9 +145,29 @@ def synchronize_resultset
         end
       end
 
-      # Clear out the previously cached .resultset
-      def clear_resultset
-        @resultset = nil
```

Comment:
resultset was only ever called in library code right after clearing it, so the whole clear and cache mechanism wasn't much good (unless I missed something)